src/shared/sentry/sentryadapter.cpp

```diff
@@ -205,3 +206,19 @@ void SentryAdapter::captureQMLStacktrace(const char* description) {
   Q_UNUSED(description);
 #endif
 }
+
+void SentryAdapter::setPlatformTag() const {
```

Comment:
Just curious why this is needed, because the Sentry event dashboard already has the platform info on the right, in the Tag Details section.
For example, for Windows it has something like:
os Windows 10.0.22621
os.name Windows
release 2.16.0